src/renderers/shared/reconciler/ReactChildReconciler.js

```diff
@@ -72,7 +71,7 @@ var ReactChildReconciler = {
    */
   updateChildren: function(
     prevChildren,
-    nextNestedChildrenElements,
+    nextChildren,
```

Comment:
The new name is inconsistent with the docs block above.  Personally I like the more explicit name `nextFlattenedChildrenElements` and would like to see that used everywhere, but at the very least they should probably be consistent?